### DISCUSSION TITLE
更新日志 - 20230609

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#824] [Announcements]
# 系统：
1. 增加部分空包以兼容某些第三方软件源，同时避免小白用户重复提交反馈  #342 #396 #822
   其中 kmod-nft-compat,  iptables-mod-socket 将集成到固件里；luci-lua-runtime 会在安装线上插件时自动解决

# 设备：
* 新增 RK3588 支持: 
  1. 增加22.03.4测试固件，支持 R6S/R6C（非此固件，点此查看 [详情](https://github.com/istoreos/istoreos/issues/716)）

**Comment by popalanluck** [comment]
R68S使用 istoreos-21.02.3-2023052618没什么问题。更新到20230609，增加了nft相关模块，导致iptables出错，shellclash无法使用tproxy模式。已经恢复出厂，并重刷0526固件。

4.19老内核固件能不能删除nft，把这些nft相关模块集成到新的22.03.4固件里，让nft与firewall4配合使用？

**Reply by jjm2473** [reply]
OK，那我把它搞成空包吧。

**Comment by mycastlex** [comment]
我想我发现上次更新0421后还掉lan口的那位朋友的问题应该是出在ssrplus+或者passwall上了，用are-u-ok的离线包安装用shadowsocks是不会掉线的，但是手动升级ssrplus后会掉，内核原因有些依赖也装不上（有可能装上了也会掉），用passwall或者ssrplus里面的xray+ss协议也会掉（我没试过别的协议，大家可以试试）。还有r2s做旁路由时重启主路由偶尔也会导致r2s掉lan口。

**Reply by qis90** [reply]
我的r5s也是升级passwall之后开始掉的，还会无限重启，旁路由模式，只有拔掉网线等启动好了才能正常使用，后来恢复系统重新装插件就好了

**Comment by bleachado** [comment]
我的上一个版本和前面一些时间，不记得版本号了，大概3月这样的固件，不知道什么情况，总是用10天左右就会断网，看它有自动重拔号，但是怎么都拔不上，要自己手机重启一下网口！机器是众筹版本的J4125 i210的

**Reply by bleachado** [reply]
我的情况应该也是和上面一位机友的一样，我现在设置了每天重启网口，试一段时间看看！

**Comment by kueU** [comment]
r68s，更新609 wan口经常掉，电信那边没查到线路问题，不知道是不是版本问题。。。

**Reply by jjm2473** [reply]
那就回退版本试试啊，或者恢复出厂设置，只配置拨号看看有没有问题。

**Reply by wtao622** [reply]
我也遇到了相同问题，现退回052311版本，目前测试了两天无问题。

**Reply by kueU** [reply]
如何回退版本，保存配置呢

**Reply by jjm2473** [reply]
直接下载上一版本固件，保留配置刷就行了

**Comment by kueU** [comment]
另外，istore商店Alist啥时候可以更新呀

**Reply by jjm2473** [reply]
你可以自己pr更新，提交到这个仓库就行了 https://github.com/linkease/istore-repo

**Comment by withhughie** [comment]
istore os拨号是不是无法跑满千兆宽带（最高大概650Mbps），Google搜索发现别人也遇到了这个问题，通过换其他家固件跑满了，希望可以支持千兆宽带

**Reply by withhughie** [reply]
设备是r66s 2.5g网口

**Reply by qis90** [reply]
进光猫后台把QOS都关了，我记得不止一处，之前我和你一样情况

**Reply by leige87** [reply]
我用的其实不光不能跑满带宽，而且性能不强，多客户端的时候ping路由都有一定的小延迟不超几十毫秒，但局域网内Ping的话应该是3以内才正常吧，一般都是纯1

**Comment by maxguy66** [comment]
这个版本更新后R5S设备开始频繁掉盘，用3.5寸硬盘一直掉，换了2.5寸硬盘+独立供电的hub，还是掉盘，最多一天就掉，最快几分钟就掉

**Reply by maxguy66** [reply]
找到掉盘的原因了，因为之前istoreos把硬盘分出来一个16MB的小分区，更新系统后，这个小分区需要格式化一下，否则就会频繁掉盘

**Reply by jjm2473** [reply]
一般是电流不够

**Reply by TheStormOfTang** [reply]
宝总目前x86的istore 内核能不能升级一下，目前关于广告拦截的插件装不上
